packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -214,6 +254,17 @@ export default class RunPanelStore {
     @action.bound
     onBotRunningEvent() {
         this.has_open_contract = true;
+        if (this.run_limit !== '') {
+            this.run_limit -= 1;
+            if (this.run_limit < 0) {
+                this.onStopButtonClick();
+                // should pass reach max run limit message to journal
+            }
+            if (this.total_profit < -this.max_losses) {
+                this.onStopButtonClick();
+                // we need a backend api to check today total lost isntread of max_losses.this function should update
+            }
```

Comment:
There is no need for us to check whether the limit has been reached, as this is enforced on the backend. Once the user has reached their limit, an error will be returned by the API after trying to purchase another proposal.

Reply:
> There is no need for us to check whether the limit has been reached, as this is enforced on the backend. Once the user has reached their limit, an error will be returned by the API after trying to purchase another proposal.

max_losses limitation has been removed
Done